
This disables SDIO PWRCTRL as fuctionality seems to be speciffic to
BSP direct sdio-soc integration...


diff --speed-large-files --no-dereference --minimal -Naur linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_bsp/Makefile linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_bsp/Makefile
--- linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_bsp/Makefile	2024-01-02 18:39:47.824581318 +0100
+++ linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_bsp/Makefile	2024-01-02 18:37:53.181247617 +0100
@@ -1,5 +1,5 @@
 CONFIG_SDIO_SUPPORT := y
-CONFIG_SDIO_PWRCTRL := y
+CONFIG_SDIO_PWRCTRL := n
 #CONFIG_AIC_FW_PATH = "/vendor/etc/firmware"
 CONFIG_AIC_FW_PATH = "/lib/firmware/aic8800"
 export CONFIG_AIC_FW_PATH
diff --speed-large-files --no-dereference --minimal -Naur linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_fdrv/Makefile linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_fdrv/Makefile
--- linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_fdrv/Makefile	2024-01-02 18:39:47.827914652 +0100
+++ linux-6.7-rc8/drivers/net/wireless/aic8800/aic8800_fdrv/Makefile	2024-01-02 18:38:06.471247654 +0100
@@ -68,7 +68,7 @@
 CONFIG_RFTEST=y
 CONFIG_USB_BT =y
 CONFIG_USE_5G ?= y
-CONFIG_SDIO_PWRCTRL ?= y
+CONFIG_SDIO_PWRCTRL ?= n
 CONFIG_CREATE_TRACE_POINTS = n
 CONFIG_TXRX_THREAD_PRIO = y
 # CONFIG_COEX = n for BT_ONLY, CONFIG_COEX =y for combo and sw
